tests/cnf/core/network/metallb/tests/bgp-gracefulrestart.go

```diff
@@ -205,6 +373,59 @@ func gracefulRestartTest(
 	Expect(err).ToNot(HaveOccurred(), "Unexpectedly succeeded in curling the Nginx pod")
 }
 
+func gracefulRestartTestWithBFD(
+	extTrafficPolicy corev1.ServiceExternalTrafficPolicyType,
+	destIPaddress,
+	ipv4metalLbIP string,
+	frrPod *pod.Builder,
+	ipAddressPool *metallb.IPAddressPoolBuilder,
+	workerLabelMap map[string]string) {
+	By("Creating a MetalLB service")
+	setupMetalLbService(netparam.IPV4Family, ipAddressPool, extTrafficPolicy)
+
+	By("Checking that routes have been added to the BGP table")
```

Comment:
to which BGP table we talk about?

Reply:
"show ip bgp"

Reply:
I mean is that the external BGP peer, right?

Reply:
right. Changed desription